dwh/indicator_lib.py

```diff
@@ -154,3 +156,33 @@ def calc_TX_PVLS(patient_agg_obs: DataFrame, VL_code: str,
   VL_agg_P['ratio'] = VL_agg_P['count']/num_patients
   return VL_agg_P
 
+def calc_TX_CURR(patient_agg_obs: DataFrame, ART_code: str,
```

Comment:
Suggestion: Let's move this new indicator and its related changes to a separate PR and keep this PR to docker related changes only (I have not looked at the details of this indicator calculation logic)..

Reply:
Noted!

Reply:
So did you forget about this or decided against it?

Reply:
Apologies, this was forgotten! Removing it now!